### DISCUSSION TITLE
Copilot update error

### DISCUSSION THREAD

**Discussion by guilhermescdev** [community/community#19077] [Copilot Conversations] [answered]
### The following error appeared when I updated the copilot:
![image](https://user-images.githubusercontent.com/84539587/174877258-be041a02-dfd9-4342-8627-57818b80bf32.png)

### When I click "Show Output log", open the terminal and this appears:
![image](https://user-images.githubusercontent.com/84539587/174877671-7995820a-418c-426f-96cd-76713e3bb087.png)

**Comment by davecheney** [comment, marked as answer]
Please upgrade your copilot extension; https://github.com/github-community/community/discussions/18829

**Reply by guilhermescdev** [reply]
I didn't find this version to install, but I tried others and it continued with the same error

**Reply by guilhermescdev** [reply]
I achieved! Apparently there was something preventing the update, but I took it off.

Thanks.

**Reply by guilhermescdev** [reply]
First I found that my vscode was not up to date. I am a windows user and the system was not letting vscode update, so I reconfigured the folder where vscode was installed through security options and then I was able to update. Then I just reinstalled the copilot and it worked. Basically the problem was windows.
